README.md

```diff
@@ -57,9 +57,10 @@ The Bolt plugin for Wash provides an accessible, interactive means of investigat
 
 ## Installation and configuration
 
+1. Install [Wash](https://puppetlabs.github.io/wash/getting_started)
 1. `gem install boltwash`
-2. Get the path to the boltwash script with `gem contents boltwash`.
-3. Add to `~/.puppetlabs/wash/wash.yaml`
+1. Get the path to the boltwash script with `gem contents boltwash`.
```

Comment:
`gem contents boltwash` has some problems if multiple versions of Boltwash are installed (on my machine, it returns the `bolt.rb` path of the _earliest_ version). We could change this to `gem contents boltwash --version <latest_version>` and add a node about using `gem list boltwash` to figure out the latter. Maybe there's an easier way to do that though.

Reply:
Not sure there's an easier way to do it. Does seem to be a problem. Maybe mention that you may need to specify `--version` to get the latest version if you have multiple installed.

Reply:
Updated